src/shared/ui/Layout/Layout.tsx

```
@@ -0,0 +1,132 @@
+import { useAuthStore } from '@/entities/auth/store/authStore';
+import { useLayoutStore } from '@/shared/store/layoutStore';
+import { defaultMenuItems } from '@/shared/model/menuItems';
+import { SideMenu } from '@/shared/ui/SideMenu';
+import { UserProfile } from '@/widgets';
+import { useState, useEffect } from 'react';
+import type { ReactNode } from 'react';
+import { useNavigate } from 'react-router-dom';
+import * as styles from './Layout.css';
+
+const { Header, Content } = Layout;
+const { Title, Text } = Typography;
```

ESLint

[error] 'Text' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)